app/general/admin.py

```diff
@@ -64,6 +71,16 @@ def clean(self):
         return cleaned_data
 
 
+class DocumentFormWithFulltext(DocumentForm):
+    class Meta:
+        model = Document
+        fields = "__all__"  # noqa: DJ007
+        labels = {"document_data": _("Document full-text (used in search)")}
```

Comment:
Maybe just a tad long. What do you think about "Searchable text"?

Reply:
Hmm, technically it's not the same as searchable text, since the description is also searchable

Reply:
On my side the label has to wrap, so it doesn't feel ideal. What about "Searchable content"? We can also add additional help text, which reminds me: let's maybe just fix this in the model by setting the verbose name and help_text. That way it appears automatically, the model is better documented, and it will appear correctly if ever used in a different form.